frontend/src/components/Common/EmptyContent/index.tsx

```diff
@@ -0,0 +1,21 @@
+/* istanbul ignore file */
```

Comment:
don't ignore the whole file, if it is not for bug bash

Reply:
will coverage next time , and already assigned to me